mixin/alerts/sidecar.libsonnet

```diff
@@ -45,8 +45,9 @@
               summary: 'Thanos Sidecar is unhealthy.',
             },
             expr: |||
-              time() - max by (%(dimensions)s) (timestamp(thanos_sidecar_last_heartbeat_success_time_seconds{%(selector)s})) >= 240
+              time() - max by (%(dimensions)s) (thanos_sidecar_last_heartbeat_success_time_seconds{%(selector)s}) >= 240
             ||| % thanos.sidecar,
+            'for': '5m',
```

Comment:
Small nit: let's remove quotes around `for`, just for consistency.

Reply:
`for` is a keyword in jsonnet and as such, it needs to be quoted.